Convex_hull_3/include/CGAL/convex_hull_3.h

```diff
@@ -690,8 +690,12 @@ ch_quickhull_polyhedron_3(std::list<typename Traits::Point_3>& points,
       non_coplanar_quickhull_3(points, tds, traits);
       copy_face_graph(tds,P);
     }
-    else
-      make_tetrahedron(v0->point(),v1->point(),v2->point(),v3->point(),P);
+    else{
+      CGAL_assertion( traits.has_on_positive_side_3_object()(
```

Comment:
`Orientation_3` is not in the traits concept and it does not matter since in case we use a kernel with exact predicates, the plane is represented as a triple of points and `Orientation_3` is then used internally.